From 13b4ecc441ad7088d67ac0116d012b8f051b0031 Mon Sep 17 00:00:00 2001
From: Han Xu <han.xu@nxp.com>
Date: Fri, 26 Oct 2018 16:29:24 -0500
Subject: [PATCH 4939/5242] MLK-20106: arm64: dts: enable NAND CE1 IOMUX for
 i.MX8MM

commit  d9b938d21b5e866b08d0160b64fde00157a10ccc from
https://source.codeaurora.org/external/imx/linux-imx.git

some NAND chips use two CS such as MT29F64G08AFAAA, which require two
enable both CE setting in IOMUX, otherwise the data may write to wrong
pages.

Signed-off-by: Han Xu <han.xu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8mm-ddr4-evk.dts     |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-ddr4-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-ddr4-evk.dts
index 5255068..1e34598 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-ddr4-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-ddr4-evk.dts
@@ -17,6 +17,7 @@
 			fsl,pins = <
 				MX8MM_IOMUXC_NAND_ALE_RAWNAND_ALE		0x00000096
 				MX8MM_IOMUXC_NAND_CE0_B_RAWNAND_CE0_B		0x00000096
+				MX8MM_IOMUXC_NAND_CE1_B_RAWNAND_CE1_B		0x00000096
 				MX8MM_IOMUXC_NAND_CLE_RAWNAND_CLE		0x00000096
 				MX8MM_IOMUXC_NAND_DATA00_RAWNAND_DATA00		0x00000096
 				MX8MM_IOMUXC_NAND_DATA01_RAWNAND_DATA01		0x00000096
-- 
1.7.9.5

